File: IQ/Classes/Ajax/clsAjaxIncrementalImportRequest.vb
Imports System.Net.Http

Public Class clsAjaxIncrementalImportRequest
    Inherits HttpRequestMessage

    Public Property lid As UInt64
    Public Property elid As UInt64
    Public Property SKUList As String
    Public Property atPoint As Int32

    Public Property SubmitList As List(Of System.Collections.Generic.KeyValuePair(Of Integer, Boolean))

End Class

File: IQ/Classes/Ajax/clsAjaxIncrementalImportRequest.cs
using System.Net.Http;

public class clsAjaxIncrementalImportRequest : HttpRequestMessage
{

	public UInt64 lid {
		get { return m_lid; }
		set { m_lid = Value; }
	}
	private UInt64 m_lid;
	public UInt64 elid {
		get { return m_elid; }
		set { m_elid = Value; }
	}
	private UInt64 m_elid;
	public string SKUList {
		get { return m_SKUList; }
		set { m_SKUList = Value; }
	}
	private string m_SKUList;
	public Int32 atPoint {
		get { return m_atPoint; }
		set { m_atPoint = Value; }
	}
	private Int32 m_atPoint;

	public List<System.Collections.Generic.KeyValuePair<int, bool>> SubmitList {
		get { return m_SubmitList; }
		set { m_SubmitList = Value; }
	}
	private List<System.Collections.Generic.KeyValuePair<int, bool>> m_SubmitList;

}
